src/Turtle/Prelude.hs

```diff
@@ -128,6 +128,13 @@ module Turtle.Prelude (
     , date
     , datefile
     , touch
+    , chmod
```

Comment:
It's probably worth giving `chmod` its own section since it introduces so many new names